src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -1486,10 +1490,8 @@ export const ComputeModalBase = ({
             existingRuntime?.tool === 'RStudio' ? h(SaveFilesHelpRStudio) : h(SaveFilesHelp)
           ])],
           [willRequireDowntime(), () => h(Fragment, [
-            existingRuntime && existingRuntime.tool !== desiredTool ?
-              p(['By continuing, you will be changing the application of your cloud environment from ', strong([existingRuntime?.tool]), ' to ',
-                strong([desiredTool]), '.']) :
-              undefined,
+            existingRuntime && existingRuntime.tool !== desiredTool ? p(['By continuing, you will be changing the application of your cloud environment from ', strong([existingRuntime.tool]), ' to ',
```

Comment:
This new code should match the previously used multiline code style for

condition ?
  trueCase :
  falseCase

to avoid a very long single line of code like we have here.

Reply:
Fix is https://github.com/DataBiosphere/terra-ui/pull/3534/files here!

Reply:
Fixed on https://github.com/DataBiosphere/terra-ui/pull/3535/files